templates/internal/appName/rpc/rpc.go.tpl

```diff
@@ -158,18 +158,18 @@ func (gs *GRPCService) StartServers(ctx context.Context, servers *Servers, opts.
 
 		{{- end }}
 	// end gRPC server option initialization injected by modules
-	{{- end }}
 
```

Comment:
Whitespace is behind the `{{- end }}` now so we only render it if we had stuff to write.